File: trunk/C#/Code/SPTemplate.UI/WebParts/SampleWebPart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.SharePoint;
using System.Web.UI;
using $customUsing$.UI.WebControls;
using $customUsing$.Common;

namespace $safeprojectname$.WebParts
{
    public class SampleWebPart : BaseWebPart
    {
        #region WebPart Methods
        protected override void CreateChildControls()
        {
            base.CreateChildControls();
        }

        protected override void OnPreRender(EventArgs e)
        {
            base.OnPreRender(e);
        }

        protected override void RenderWebPart(HtmlTextWriter output)
        {
            base.RenderWebPart(output);
        }

        #endregion
    }
}

File: VB/Sample Project/SampleProject/SampleProject.UI/WebParts/SampleWebPart.vb
Imports System.Web.UI

Namespace WebParts
    Public Class SampleWebPart
        Inherits BaseWebPart
#Region "WebPart Methods"
        Protected Overrides Sub CreateChildControls()
            MyBase.CreateChildControls()
        End Sub

        Protected Overrides Sub OnPreRender(ByVal e As EventArgs)
            MyBase.OnPreRender(e)
        End Sub

        Protected Overrides Sub RenderWebPart(ByVal output As HtmlTextWriter)
            MyBase.RenderWebPart(output)
        End Sub

#End Region
    End Class
End Namespace
